src/decision/dMCDA.jl

```diff
@@ -604,53 +631,68 @@ function guided_site_selection(
     # if seeding, create seeding specific decision matrix
     if log_seed
         SE, wse = create_seed_matrix(
-            A, d_vars.min_area, d_vars.wt_in_conn_seed, d_vars.wt_out_conn_seed,
-            d_vars.wt_waves, d_vars.wt_heat, d_vars.wt_predec_seed, d_vars.wt_zones_seed,
-            d_vars.wt_lo_cover
+            A,
```

Comment:
Maybe I did this in the mcda branch but might be more consistent to have every input on a new line, or 2 lines of the same length for inputs instead of some on a new line and some on the same line.